ossdbs/fem/volume_conductor/volume_conductor_model.py

```diff
@@ -244,7 +244,10 @@ def run_full_analysis(
                     self._impedances[band_indices] = impedance
 
                 # refine only at first frequency
-                if freq_idx == frequency_indices[0] and adaptive_mesh_refinement:
+                if (
+                    freq_idx == frequency_indices[0]
+                    and adaptive_mesh_refinement["Active"]
```

Comment:
Is this safe? Isn't `adaptive_mesh_refinement`  bool in most of the cases?

Reply:
Yes, the structure of the settings for adaptive mesh refinement has been slightly modified. Now it's a dictionary, where you can set it as ´True´ and ´False but also modify MaxIterations and the error.